style.css

```diff
@@ -0,0 +1,588 @@
+}
+.email-input input {
+    border: none;
+    outline: none;
+    width: 100%;
+}
+.email-input input::placeholder {
+    opacity: 0.3;
+}
+.email-wrapper input {
+    border-radius: 0;
+}
+.intro-img {
+    float: left;
+    width: 40%;
+}
+.intro-img img {
+    max-width: 100%;
+}
+.compatible-icons {
+    display: flex;
+    justify-content: space-between;
+    width: 421px;
+    opacity: .3;
+}
+.compatible-icons svg path,
+.compatible-icons svg g {
+    fill: gray;
+}
+
+/* features-section */
+.features-section {
+    text-align: center;
+}
+.features-info h2 {
+    font-size: 60px;
+    max-width: 600px;
+    margin-left: auto;
+    margin-right: auto;
+    font-weight: 400;
+    margin-bottom: 20px;
+}
+.features-info p {
+    max-width: 610px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 80px;
+}
+.customized-items {
+    display: flex;
+    justify-content: space-between;
+}
+.customized-items h4 {
+    font-size: 18px;
+}
+.customized-items p {
+    opacity: 0.7;
+    max-width: 300px;
+}
+.customized-item-icon {
+    background: black;
+    width: 50px;
+    height: 50px;
+    border-radius: 14px;
+    margin-left: auto;
+    margin-right: auto;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.learn-more {
+    color: #540CFA;
+}
+.customized-item-machine-learning .customized-item-icon {
+    background: linear-gradient(135deg, #F2C314,#FC3832);
+}
+.customized-item-automated-tracking .customized-item-icon {
+    background: linear-gradient(135deg, #9C09DB,#1C0FD7);
+}
+.customized-item-smart-notifications .customized-item-icon {
+    background: linear-gradient(135deg, #32FBFC,#3214F2);
+}
+.movie-slides {
+    display: flex;
+    padding: 140px 0;
+}
+
+/* interface section */
+.interface-intro h2 {
+    font-size: 46px;
+    font-weight: 400;
+    max-width: 460px;
+    line-height: 1;
+}
+.interface-intro p {
+    max-width: 460px;
+    margin-bottom: 57px;
+}
+.interface-intro-btn {
+    display: flex;
+    align-items: center;
+    margin-bottom: 20px;
+    padding: 17px;
+    max-width: 340px;
+    opacity: 0.5;
+    transition: all .2s ease-in-out;
+    border-radius: 4px;
+    font-size: 16px;
+}
+
+.interface-intro-btn svg {
+    margin-right: 20px;
+    width: 20px;
+    height: 20px;
+}
+.interface-intro-btn:hover {
+    opacity: 1;
+    box-shadow: rgba(0, 0, 0, 0.1) 0 1px 3px 0, rgba(0, 0, 0, 0.05) 0 1px 2px 0;
+}
+.interface-intro {
+    float: left;
+    width: 42%;
+    margin-right:9%
+}
+.interface-img {
+    float: left;
+    width: 49%;
+    position: relative;
+}
+.interface-img-iphone {
+    position: relative;
+    z-index: 2;
+    left: 0;
+    margin-top: 100px;
+}
+.interface-img-web {
+    position: absolute;
+    top: -48px;
+    left: 88px;
+    z-index: 1;
+}
+.interface-section {
+    overflow: hidden;
+    padding-bottom: 100px;
+}
+
+/* community */
+.community-section {
+    color: white;
+    background-image: url(testimonials.png);
```

Comment:
This path needs to be updated to fix missing background:

<img width="1318" alt="Screen Shot 2020-04-18 at 9 02 31 PM" src="https://user-images.githubusercontent.com/10561050/79648047-176a5980-81b9-11ea-8ee6-9d2830239be1.png">